### PR TITLE
q46

### PR DESCRIPTION
**Pull request by brar-navleen**
q46:
image

### CODE REVIEW

File: Javascript/q46.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  </style>
+
+  <script>
+    function myFunction() {
+      const a = Number(document.querySelector("#demo0").value);
+      const b = Number(document.querySelector("#demo1").value);
+      const c = document.querySelector("#demo2").value;
+
+      switch(c){
+      case "+": document.getElementById("demo3").innerHTML= a+b;
```
Comment (Mantaseus) [inline Javascript/q46.html:28]: Please do proper indentation in this switch case block and take a look at the clean way of defining switch case blocks. It usually goes like this

```js
switch(c) {
  case '+':
    // Do something
    break;
}
```

File: Javascript/q46.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+    .flex {
+      display: flex;
+      gap: 10px;
+      flex-wrap: wrap;
+    }
+  </style>
+
+  <script>
+    function myFunction() {
```
Comment (Mantaseus) [inline Javascript/q46.html:22]: Please use a better name for the function

File: Javascript/q46.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+      const a = Number(document.querySelector("#demo0").value);
```
Comment (Mantaseus) [inline Javascript/q46.html:23]: What is `a`, `b`, and `c`? Please improve their names. Please

File: Javascript/q46.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+      break;
+      case "/": document.getElementById("demo3").innerHTML= a/b;
+    }
+  }
+  </script>
+</head>
+
+<body>
+  <div class="flex">
+    <input id="demo0" type="number">
```
Comment (Mantaseus) [inline Javascript/q46.html:42]: Please use better ID values

File: Javascript/q46.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  <script>
+    function myFunction() {
+      const a = Number(document.querySelector("#demo0").value);
+      const b = Number(document.querySelector("#demo1").value);
+      const c = document.querySelector("#demo2").value;
+
+      switch(c){
+      case "+": document.getElementById("demo3").innerHTML= a+b;
+      break;
+      case "*": document.getElementById("demo3").innerHTML= a*b;
```
Comment (Mantaseus) [inline Javascript/q46.html:30]: It is customary to keep space around the match operations. Please fix it in the other cases as well. `a * b`

File: Javascript/q46.html (hunk 1 above)
Comment (Navleenkaursandhu) [reply Javascript/q46.html:28]: ok

File: Javascript/q46.html (hunk 2 above)
Comment (Navleenkaursandhu) [reply Javascript/q46.html:22]: done

File: Javascript/q46.html (hunk 4 above)
Comment (Navleenkaursandhu) [reply Javascript/q46.html:42]: done

File: Javascript/q46.html (hunk 5 above)
Comment (Navleenkaursandhu) [reply Javascript/q46.html:30]: yes done

**Pull request merged**